### PR TITLE
CHANGELOG-3.2: update from #7892

### PR DESCRIPTION
**Pull request by wenjiaswe**
CHANGELOG-3.2: update from #7892

### CODE REVIEW

File: CHANGELOG-3.2.md [hunk 1]
```diff
@@ -28,7 +28,7 @@ Note that any `etcd_debugging_*` metrics are experimental and subject to change.
 - Add [`etcd_mvcc_db_total_size_in_bytes`](https://github.com/coreos/etcd/pull/9819) Prometheus metric.
   - In addition to [`etcd_debugging_mvcc_db_total_size_in_bytes`](https://github.com/coreos/etcd/pull/9819).
 - Add [`etcd_mvcc_db_total_size_in_use_in_bytes`](https://github.com/coreos/etcd/pull/9256) Prometheus metric.
-  - Use it with `etcd_mvcc_db_total_size_in_bytes` and `etcd_mvcc_db_total_size_in_use_in_bytes`.
+  - Use it with `etcd_server_quota_backend_bytes` and `etcd_mvcc_db_total_size_in_use_in_bytes`.
```
Comment (gyuho) [inline CHANGELOG-3.2.md]: revert this change? We already have this change log in line 22?

**Comment by gyuho** [comment]
I think we need the same change from https://github.com/coreos/etcd/commit/a47f0a0dbadeb9a775189a1153283e956e8cc7ec for 3.2 changelog?

**Comment by gyuho** [comment]
@wenjiaswe Ping? Once we update this, we are ready for patch release. Thanks.

File: CHANGELOG-3.2.md (hunk 1 above)
Comment (wenjiaswe) [reply CHANGELOG-3.2.md]: It is kinda repeating with line 22. We could either remove line 31 completely, or I think we were trying to say use the three metrics (etcd_mvcc_db_total_size_in_use_in_bytes, etcd_server_quota_backend_bytes and etcd_mvcc_db_total_size_in_bytes) together, before it mentioned etcd_mvcc_db_total_size_in_use_in_bytes again but didn't say etcd_server_quota_backend_bytes. Well, apparently I changed the wrong thing though, I meant to s/etcd_mvcc_db_total_size_in_use_in_bytes/etcd_server_quota_backend_bytes

Comment (gyuho) [reply CHANGELOG-3.2.md]: Oh, I just noticed that. Yeah they are redundant.

**Comment by wenjiaswe** [comment]
@gyuho Sorry missed this review. I just updated for line 31, with the correct replacement, please let me know if that's OK or not. For ["the same change from a47f0a0 for 3.2 changelog"](the same change from a47f0a0 for 3.2 changelog), I delivered in #9943 .

**Comment by gyuho** [comment]
@wenjiaswe Can you also help merge https://github.com/coreos/etcd/pull/9950? So we do not do this kind of manual checking? Thanks!

**Comment by wenjiaswe** [comment]
Yes, will do!
